front/src/modules/ui/board/components/EntityBoard.tsx

```diff
@@ -104,6 +108,23 @@ export function EntityBoard({
 
   const boardRef = useRef<HTMLDivElement>(null);
 
+  useEffect(() => {
+    function handleKeyUp(event: KeyboardEvent) {
+      if (event.key === 'Escape') {
+        unselectAllActiveCards();
+      }
+    }
+    window.addEventListener('keyup', handleKeyUp);
+    return () => {
+      window.removeEventListener('keyup', handleKeyUp);
+    };
+  }, [unselectAllActiveCards]);
+
+  useListenClickOutside({
+    refs: [boardRef],
```

Comment:
I'm using `useListenClickOutside` for clicker events. However, notion software looks like if you click any space board area - board entity card, it should unselect. I would normally pass in the card refs, but as you can see card refs already exist in that component scope. 

What are your thoughts @lucasbordeau?

Reply:
What would really be great here is to have a useListenClickOutside that takes a classname instead of a refs array. Then by adding a classname on entity board card it would be easy to listen for any click outside all cards.

Do you feel ok with working on this ?

Reply:
Here's a draft of the solution : 
- add a data-click-outside-id attribute on the entity board card
- create a new useListenClickOutsideId hook (we'll refactor later) that does the same but takes a string id instead of refs
- the hook will test that the clicked element has the data attribute data-click-outside-id with the same id
- use this hook here

Reply:
Ah gotcha -- I'm on it!